tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -377,21 +377,29 @@ def test_count_transaction_with_invalid_condition(self):
                 use_case_id=UseCaseKey.PERFORMANCE,
             )
 
-    @freeze_time()
+    @freeze_time("2022-09-22 11:07:00")
```

Comment:
this line only helps to reproduce flakiness in all situations. Also it helps to demonstrate that fix do the job.
If you try this value on master branch you can see failing. Also if you change time to `("2022-09-22 11:07:01")` it starts working in master branch that's why it''s flaky.